### PR TITLE
Flightpaths

### PR DESCRIPTION
**Pull request by IntelligentQuantum**
## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/2816

@BarbzYHOOL
Tested in game, But problem is here you can't see right Price in flight master

### CODE REVIEW

**Comment by Meltie2013** [comment]
@IntelligentQuantum The possible issue is that you are missing changes in `PlayerTaxi.cpp`. There is price entries there when the data is being loaded/called.  I am not 100% sure on this, maybe something to look into to get the correct pricing to display in the game.
https://github.com/azerothcore/azerothcore-wotlk/blob/4d4c902cdad2d7decda91a869feaa28f4a74addf/src/server/game/Entities/Player/PlayerTaxi.cpp#L106-L140

https://github.com/azerothcore/azerothcore-wotlk/blob/4d4c902cdad2d7decda91a869feaa28f4a74addf/src/server/game/Entities/Player/PlayerTaxi.cpp#L156-L167

**Comment by IntelligentQuantum** [comment]
> @IntelligentQuantum The possible issue is that you are missing changes in `PlayerTaxi.cpp`. There is price entries there when the data is being loaded/called. I am not 100% sure on this, maybe something to look into to get the correct pricing to display in the game.
> 
> https://github.com/azerothcore/azerothcore-wotlk/blob/4d4c902cdad2d7decda91a869feaa28f4a74addf/src/server/game/Entities/Player/PlayerTaxi.cpp#L106-L140
> 
> https://github.com/azerothcore/azerothcore-wotlk/blob/4d4c902cdad2d7decda91a869feaa28f4a74addf/src/server/game/Entities/Player/PlayerTaxi.cpp#L156-L167

Thanks, i will check it.

**Review by Helias — APPROVED** [review]

File: src/server/game/World/World.cpp [hunk 1]
```diff
@@ -1405,6 +1405,13 @@ void World::LoadConfigSettings(bool reload)
     // Specifies if IP addresses can be logged to the database
     m_bool_configs[CONFIG_ALLOW_LOGGING_IP_ADDRESSES_IN_DATABASE] = sConfigMgr->GetOption<bool>("AllowLoggingIPAddressesInDatabase", true, true);
 
+    rate_values[RATE_FLIGHT_PATH_COST]  = sConfigMgr->GetOption<float>("Rate.Flightpaths.Cost", 1.0f);
+    if (rate_values[RATE_FLIGHT_PATH_COST] < 0.0f)
+    {
+        sLog->outError("Rate.Flightpaths.Cost (%f) must be >=0. Using 0.0 instead.", rate_values[RATE_FLIGHT_PATH_COST]);
+        rate_values[RATE_FLIGHT_PATH_COST] = 0.0f;
```
Comment (Kitzunu) [inline src/server/game/World/World.cpp:1412]: Should be 1 instead. x * 0 = 0 so would remove the cost

**Comment by FrancescoBorzi** [comment]
what's the status of this? WIP? ready to be reviewed? waiting to be tested?

please apply labels accordingly

**Pull request closed without merging**